scripts/deploy-helm.sh

```diff
@@ -341,26 +327,12 @@ if [[ $DEPLOYMENT_TYPE == all ]] || [[ $DEPLOYMENT_TYPE == metrics ]]; then
 
   # If --without-extensions-upgrade is set, all gcp extensions are downloaded from the cluster to get configuration of gcp services for versions that are currently active on the cluster.
   if [[ "$UPGRADE_EXTENSIONS" == "N" && -n "$EXTENSIONS_FROM_CLUSTER" ]]; then
-    debug "Downloading activated extensions from Dynatrace environment"
     info ""
-    info "- downloading active extensions from Dynatrace"
-    get_extensions_from_dynatrace "$EXTENSIONS_FROM_CLUSTER"
+    info "- extensions were already activated in Dynatrace"
```

Comment:
can we make it little more explicit?
"Some xtensions are already activated in Dynatrace. No new extensions will be activated."